geode-junit/src/main/java/org/apache/geode/test/version/VersionManager.java

```diff
@@ -126,6 +122,25 @@ public String getInstall(String version) {
     return installs.get(version);
   }
 
+  /**
+   * Remove the dots from a version string. "1.2.0" -> "120"
+   */
+  public String versionWithNoDots(String s) {
+    StringBuilder b = new StringBuilder(10);
+    int length = s.length();
+    for (int i = 0; i < length; i++) {
+      char ch = s.charAt(i);
+      if (ch != '.') {
+        // leave off any trailing stuff like "-incubating"
+        if (!Character.isDigit(ch)) {
+          break;
+        }
+        b.append(s.charAt(i));
```

Comment:
Can you just use the `ch` variable here instead of calling `s.charAt(i)` again?

Reply:
okay - I'll change that